src/helpers.ts

```diff
@@ -77,3 +77,28 @@ export function setBody(body: string) {
 export function setNamespace(namespace: string) {
   setAttribute("appsignal.namespace", namespace)
 }
+
+export function setError(error: Error) {
+  if (error && error.name && error.message) {
+    const activeSpan = trace.getActiveSpan()
+    if (activeSpan) {
+      activeSpan.recordException(error)
+      activeSpan.setStatus({
+        code: SpanStatusCode.ERROR,
+        message: error.message
+      })
+    }
+  }
+}
+
+export function sendError(error: Error, fn: () => void = () => {}) {
+  if (error && error.name && error.message) {
+    trace
+      .getTracer("Appsignal.sendError")
```

Comment:
If this name determines the "instrumentationLibrary" name, we may want to create an extractor for it, or at least not have it warn about a missing extractor.

Reply:
I believe that, since it will always create a root span, it should not have a visible warning about a missing extractor. But happy to add an extractor for it.

Reply:
I've checked the extractor and I'm not sure what would need to change from the default handling. Let me know your thoughts on what the extractor should do.

Reply:
It looks better than I thought. It doesn't say anything about a missing incident action name. The other warnings only really show on performance samples in the event timeline.